src/overlays/actors/ovl_En_Dt/z_en_dt.h

```diff
@@ -2,16 +2,42 @@
 #define Z_EN_DT_H
 
 #include "global.h"
+#include "objects/object_dt/object_dt.h"
+
+#define EN_DT_CUTSCENE_COUNT 11
 
 struct EnDt;
 
 typedef void (*EnDtActionFunc)(struct EnDt*, PlayState*);
 
 typedef struct EnDt {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0xF8];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[OBJECT_DT_LIMB_MAX];
+    /* 0x1E2 */ Vec3s morphTable[OBJECT_DT_LIMB_MAX];
     /* 0x23C */ EnDtActionFunc actionFunc;
-    /* 0x240 */ char unk_240[0xA0];
-} EnDt; // size = 0x2E0
+    /* 0x240 */ s32 showedCouplesMask; // Is set, but never read from
+    /* 0x244 */ s16 timer;
+    /* 0x246 */ s16 animIndex;
+    /* 0x248 */ s16 disableBlinking;
+    /* 0x24A */ s16 blinkTimer;
+    /* 0x24C */ s16 eyeTexIndex;
+    /* 0x24E */ // compiler padding
+    /* 0x250 */ f32 animEndFrame;
+    /* 0x254 */ s16 npcState;
+    /* 0x256 */ s16 textIdIndex;
+    /* 0x258 */ s16 csIds[EN_DT_CUTSCENE_COUNT];
+    /* 0x26E */ s16 csIdIndex;
+    /* 0x270 */ s16 cutsceneState;
+    /* 0x272 */ // compiler padding
```

Comment:
you could add an `UNK_TYPE1 unk272` // struct padding comment?

Reply:
Is that a standard? I couldn't find anything in the docs specifying one way or another, and if the goal is to match original source as closely as possible, it seems strange to introduce unused fields that are inserted automatically by the compiler for alignment.

Reply:
Generally we don't even document it and leave it implicit. We just remove the entry along with the offset comment.

Reply:
Understood, that makes sense to me.